src/backend/mailbox_manager/migrations/0018_fix_mailboxes_status.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
+
+from mailbox_manager import enums
+
+
+def change_some_mailboxes_status_to_enabled(apps, schema_editor):
+    Mailbox = apps.get_model('mailbox_manager', 'Mailbox')
+    Mailbox.objects.filter(
+        status=enums.MailboxStatusChoices.PENDING,
+        domain__name__in=["mail.numerique.gouv.fr", "data.gouv.fr"]
```

Comment:
I wonder if domain names in clear poses a security issue or not at all

Reply:
No but it's too specific. I will send a script to Jacques ;)